examples/flask/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "sgid-flask-app"
+version = "0.1.0"
+description = ""
+authors = ["Open Government Products"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+flask = "^2.3.2"
+requests = "^2.30.0"
+python-dotenv = "^1.0.0"
+jwcrypto = "^1.4.2"
+sgid-client = {git = "https://github.com/opengovsg/sgid-client-python.git", rev = "feat/init-build"}
```

Comment:
this can't be updated until we publish though. that branch is the only place where we can get the built files currently. as per the release plan in https://github.com/opengovsg/sgid-client-python/pull/5, once we publish the prerelease, we can install from PyPI instead of github